tests/snapshots/TestCLICommands_Valid_log_file_in_flag_should_be_priortized_over_env_and_config.stdout.golden

```diff
@@ -1,7 +1,6 @@
+DEBU Log level set to DEBU file=/dev/stdout
```

Comment:
We should convey where this value came from or how do we know it’s working as intended?

Reply:
we know it is working as intended because of test cases as of now.